pyarn/lexer.py

```diff
@@ -35,6 +35,13 @@ def t_NUMBER(t):
     return t
 
 
+# Note: this must be defined BEFORE t_STRING, the grammar is ambiguous
+def t_BOOLEAN(t):
+    r'true|false'
```

Comment:
Should this regex string be part of something?

Reply:
oh, NVM. I need to read it more

Reply:
It's the docstring for `t_BOOLEAN`, that's how tokens are defined in [ply](https://www.dabeaz.com/ply/)